docs/docs/python/integrations/google-cloud-functions.mdx

```diff
@@ -67,6 +66,42 @@ Use [layout](../../../reference/targets/python_google_cloud_function.mdx#layout)
 `file` / `files` targets will not be included in the built Cloud Function because filesystem APIs like `open()` would not load them as expected. Instead, use the `resource` / `resources` target. See [Assets and archives](../../using-pants/assets-and-archives.mdx) for further explanation.
 :::
 
+### Specifying a runtime explicitly
+
+Pants will attempt to infer the appropriate runtime and its platform-specific details from [the relevant interpreter constraints](../overview/interpreter-compatibility.mdx). For example, given `interpreter_constaints = ["==3.12.*"]`, Pants will build the Lambda package to appropriate for the  `python312` runtime. The Lambda environment is a relatively stable, running Linux on a x86-64 CPU, and thus Pants includes default "complete platforms" specifications that allow choosing appropriate artifacts of third-party dependencies with native code. These are chosen based on the (inferred or explicit) runtime.
```

Comment:
Maybe we should use "GCF" instead of "Lambda" in this doc?

Reply:
Yes, we definitely should. You can tell which one I wrote first and copied from 😄